ibm/data_source_ibm_cm_offering_instance.go

```diff
@@ -97,6 +97,16 @@ func dataSourceIBMCmOfferingInstance() *schema.Resource {
 				Computed:    true,
 				Description: "id of the resource group",
 			},
+			"install_plan": &schema.Schema{
```

Comment:
These two attributes `install_plan` and `channel` are declared in schema.. but not set in READ method of this Datasource.